crates/orderbook/src/database/orders.rs

```diff
@@ -277,6 +277,17 @@ fn calculate_status(order: &FullOrder) -> OrderStatus {
 fn full_order_into_model_order(order: FullOrder) -> Result<Order> {
     let status = calculate_status(&order);
     let pre_interactions = extract_pre_interactions(&order)?;
+    let ethflow_data = if let Some(user_valid_to) = order.ethflow_data.1 {
```

Comment:
One suggestions to avoid the unwrap is:
```suggestion
    let ethflow_data = if let Some((is_refunded, user_valid_to)) = order.ethflow_data.0.and(order.ethflow_data.1) {
```

The reason I suggest this is that there is a bit of "logic assertion asymmetry" where if the `valid_to` is some bu the `is_refunded` is none, then we panic, but if the `valid_to` is none but the `is_refunded` is some, then we silently continue. Alternatively, if you really want to assert, then you should:
```
match (order.ethflow_data) {
    (Some(is_refunded), Some(user_valid_to)) => { ... },
    (None, None) => None,
    _ => {
        panic!("should never happen!");
    }
}
```